dpgen2/op/run_nvnmd_train.py

```
@@ -0,0 +1,670 @@
+            - `lcurve`: (`Artifact(Path)`) The learning curve file.
+            - `log`: (`Artifact(Path)`) The log file of training.
+
+        Raises
+        ------
+        FatalError
+            On the failure of training or freezing. Human intervention needed.
+        """
+        mixed_type = ip["optional_parameter"]["mixed_type"]
+        config = ip["config"] if ip["config"] is not None else {}
+        dp_command = ip["config"].get("command", "dp").split()
+        train_args = config.get("train_args", "")
```

Ruff (0.11.9)

Local variable `train_args` is assigned to but never used

Remove assignment to unused variable `train_args`

(F841)